rewrite-java/src/main/java/org/openrewrite/java/format/TabsAndIndentsVisitor.java

```diff
@@ -232,27 +234,52 @@ public Space visitSpace(Space space, Space.Location loc, P p) {
                         JContainer<J> container = getCursor().getParentOrThrow().getValue();
                         List<J> elements = container.getElements();
                         J lastArg = elements.get(elements.size() - 1);
-                        if (elements.size() > 1 && style.getMethodDeclarationParameters().getAlignWhenMultiple()) {
-                            J.MethodDeclaration method = getCursor().firstEnclosing(J.MethodDeclaration.class);
-                            if (method != null) {
-                                int alignTo = computeFirstParameterColumn(method);
-                                if (alignTo != -1) {
-                                    getCursor().getParentOrThrow().putMessage("lastIndent", alignTo - style.getContinuationIndent());
-                                    elem = visitAndCast(elem, p);
-                                    getCursor().getParentOrThrow().putMessage("lastIndent", indent);
-                                    after = indentTo(right.getAfter(), t == lastArg ? indent : alignTo, loc.getAfterLocation());
+                        //noinspection ConstantConditions
+                        J tree = null;
+                        if (loc == JRightPadded.Location.METHOD_DECLARATION_PARAMETER) {
+                            tree = getCursor().firstEnclosing(J.MethodDeclaration.class);
+                        } else if (loc == JRightPadded.Location.RECORD_STATE_VECTOR) {
+                            tree = getCursor().firstEnclosing(J.ClassDeclaration.class);
+                            getCursor().getParentOrThrow().putMessage("indentType", IndentType.CONTINUATION_INDENT);
+                        }
+                        if (elements.size() > 1) {
+                            try {
+                                if ((loc == JRightPadded.Location.METHOD_DECLARATION_PARAMETER && style.getMethodDeclarationParameters().getAlignWhenMultiple()) ||
+                                        ((loc == JRightPadded.Location.RECORD_STATE_VECTOR && style.getRecordComponents() != null && style.getRecordComponents().getAlignWhenMultiple()))) {
```

Comment:
`style.getRecordComponents()` is not marked as `@Nullable`; should it be? Or can we remove this guard since we seem to rely on the `NoSuchMethodError` below.